axum-core/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # 0.5.0
 
+- **change:** Change rejection `Display` impl to match `body_text()` result ([#3118])
```

Comment:
I think this doesn't have enough info. We need to
- mention that this affects all the (composite / enum?) extractors defined by `axum` and `axum-extra` or move this to those changelogs
- state what was different about these before / what's the user-visible change

Reply:
I've slightly tweaked it. Is that better? 

> * state what was different about these before / what's the user-visible change

I don't think there is a user-visible change in most cases since the `body_text()` impl hasn't really changed. The only thing that changed is the `Display` impl, which probably isn't actually visible to the user in most cases.

Reply:
I meant how the change would be observable by users of axum, not (necessarily) end users.

New entry looks good 👍